Cargo.toml

```diff
@@ -1,22 +1,22 @@
 [package]
 name = "phoenix-core"
-version = "0.10.0"
+version = "0.11.0-rc.0"
 authors = ["zer0 <matteo@dusk.network>", "Victor Lopez <victor@dusk.network"]
 edition = "2018"
 
 [dependencies]
-rand_core = "0.5"
+rand_core = "0.6"
 dusk-bytes = "0.1"
-dusk-bls12_381 = {version = "0.6", default-features = false}
-dusk-jubjub = {version = "0.8", default-features = false}
-dusk-poseidon = {version = "0.20", default-features = false}
-dusk-pki = {version = "0.6", default-features = false}
-canonical = {version = "0.5", optional = true}
-canonical_derive = {version = "0.5", optional = true}
+dusk-bls12_381 = {version="0.8", default-features=false}
+dusk-jubjub = {version="0.10", default-features=false}
+dusk-poseidon = {version="0.21.0-rc", default-features=false}
+dusk-pki = {version="0.7.0-rc", default-features=false}
+canonical = {version="0.6", optional=true}
+canonical_derive = {version="0.6", optional=true}
```

Comment:
Is there any reason why you remove the space around `=`?

Reply:
It was to save space and reduce long features list to a single line.

As agreed, we should leave the space. Fixed